### DISCUSSION TITLE
More city details to add

### DISCUSSION THREAD

**Discussion by rachelpr** [rachelpr/LeafSmart#94] [Ideas]
Could add details like:

- Local currency/exchange rate
- Language spoken
- Elevation
- Ride-sharing apps (if available)

**Comment by b1u3too** [comment]
I like this idea! I think some of this data is available through the teleport api, city details itinerary

**Reply by rachelpr** [reply]
Yeah I think exchange rate/currency is, but I'd have to double check

**Reply by b1u3too** [reply]
Cross posted with #100 : [Link](https://docs.google.com/document/d/1AByipwwDinR9Rxb6ogw78ijeqmQSRxuDIUQnw_suWyY/edit?usp=sharing)
